packages/apollo-federation/src/composition/__tests__/composeAndValidate.test.ts

```diff
@@ -99,23 +99,16 @@ it('composes and validates all (24) permutations without error', () => {
     reviewsService,
     accountsService,
     productsService,
-  ]).map((config, i) => {
-    const { warnings, errors } = composeAndValidate(config);
+  ]).map(config => {
+    const { errors } = composeAndValidate(config);
 
-    if (warnings.length || errors.length) {
+    if (errors.length) {
       console.error(
-        `Errors or warnings found with composition [${config.map(
-          item => item.name,
-        )}]`,
+        `Errors found with composition [${config.map(item => item.name)}]`,
```

Comment:
I'll leave as is, since the future dev who breaks one of these permutations will likely have to add their own similar logging to track down the "bad case". I was just cleaning up some linting (and warnings are no longer a thing).